helix-term/src/commands.rs

```diff
@@ -3400,17 +3403,38 @@ fn match_mode(cx: &mut Context) {
                 'm' => match_brackets(cx),
                 's' => surround_add(cx),
                 'r' => surround_replace(cx),
-                'd' => {
-                    surround_delete(cx);
-                    let (view, doc) = current!(cx.editor);
-                }
+                'd' => surround_delete(cx),
+                'a' => select_textobject(cx, textobject::TextObject::Around),
+                'i' => select_textobject(cx, textobject::TextObject::Inner),
                 _ => (),
             }
         }
     })
 }
 
-use helix_core::surround;
+fn select_textobject(cx: &mut Context, objtype: textobject::TextObject) {
```

Comment:
I would advise against adding more arguments other than `cx` for autoinfo to be able to make use of it.

Reply:
It's just a wrapper func that's called by something else that's a `command`. I do that elsewhere

Reply:
Yes, I saw it. But later I will change it.

Reply:
Yeah it's similar to how `o` and `O` is implemented:

https://github.com/helix-editor/helix/blob/6c4e1b02093608b2778a8e634d83c4042022c08b/helix-term/src/commands.rs#L1871-L1873

https://github.com/helix-editor/helix/blob/6c4e1b02093608b2778a8e634d83c4042022c08b/helix-term/src/commands.rs#L1938-L1946